suite-native/module-send/src/screens/SendOutputsScreen.tsx

```diff
@@ -210,7 +213,8 @@ export const SendOutputsScreen = ({
                 setValue('rippleDestinationTag', sendFormDraft.rippleDestinationTag, {
                     shouldTouch: true,
                 });
-                await calculateNormalFeeMaxAmount();
+                // The max amount is equal to the total token balance for tokens. (fee is paid in mainnet currency)
```

Comment:
calculations for max should not be done in case tokens are sent. This will cause error at least on solana